docs/VnVPlan/VnVPlan.tex

```diff
@@ -142,95 +148,246 @@ \subsection{Relevant Documentation}
+
+% \wss{Create a checklists?}
 
 \subsection{Implementation Verification Plan}
 
-\wss{You should at least point to the tests listed in this document and the unit
-  testing plan.}
+As part of the implementation verification plan, the team will conduct a series of static tests. One of which, will be a code walkthrough with members of the team.
+Before submitting Rev 0 and Rev 1, the members of the team will schedule time to walk through other members of the team through the code they have written. This will be 
+an opportunity for the team to look for code quality and adherence to standards, correctness, and alignment to the system design. During this walkthrough, we expect that 
+the lead developer of the code being displayed will explain the overall flow and control structures. In doing so, we expect they will find the errors in their own code.\\\\
+The team will also use a suite of tests, which is described in section 4.\\\\
+The team will used static analyzers to verify adherence to coding standards to perform type checking. Please refer to section 7.2.2 and 7.2.3 in the Development Plan found \href{https://github.com/OKKM-insights/OKKM.insights/blob/main/docs/DevelopmentPlan/DevelopmentPlan.pdf}{here}.
+
+% \wss{You should at least point to the tests listed in this document and the unit
+%   testing plan.}
 
-\wss{In this section you would also give any details of any plans for static
-  verification of the implementation.  Potential techniques include code
-  walkthroughs, code inspection, static analyzers, etc.}
+% \wss{In this section you would also give any details of any plans for static
+%   verification of the implementation.  Potential techniques include code
+%   walkthroughs, code inspection, static analyzers, etc.}
 
-\wss{The final class presentation in CAS 741 could be used as a code
-walkthrough.  There is also a possibility of using the final presentation (in
-CAS741) for a partial usability survey.}
+% \wss{The final class presentation in CAS 741 could be used as a code
+% walkthrough.  There is also a possibility of using the final presentation (in
+% CAS741) for a partial usability survey.}
 
 \subsection{Automated Testing and Verification Tools}
 
-\wss{What tools are you using for automated testing.  Likely a unit testing
-  framework and maybe a profiling tool, like ValGrind.  Other possible tools
-  include a static analyzer, make, continuous integration tools, test coverage
-  tools, etc.  Explain your plans for summarizing code coverage metrics.
-  Linters are another important class of tools.  For the programming language
-  you select, you should look at the available linters.  There may also be tools
-  that verify that coding standards have been respected, like flake9 for
-  Python.}
+Please refer to section 7.2.2 and 7.2.3 in the Development Plan found \href{https://github.com/OKKM-insights/OKKM.insights/blob/main/docs/DevelopmentPlan/DevelopmentPlan.pdf}{here}.
```

Comment:
I think we can expand on what is in the dev plan. There is no mention of actual testing frameworks such as pytest for python, maybe somethin like Jest or React test library for front end, mocking tools such as Mockito, whatever tools that would be necessary to test an ai model in some way. Also we can probably mention some code coverage tools for python and typescript too.

Reply:
That sounds great - I'll go through and add those. Are you okay to commit these changes now then?

Reply:
Oh I see, you are suggesting I add the additional info in the VnV plan. I'll make the changes now and request a review when they're done.

Reply:
Use your discretion on whatever you think would be best, it wouldn't hurt to update both the devplan tex and Vnv plan tex under this pr. Honestly, just updating the dev plan may suffice